dist/flatpak/com.github.input-leap.input-leaps.yml

```diff
@@ -10,34 +10,63 @@
 # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1013
 
 app-id: com.github.inputleap.InputLeap
-runtime: org.gnome.Platform
-runtime-version: '42'
-sdk: org.gnome.Sdk
-command: input-leap-flatpak
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: input-leap
 
 finish-args:
     # We want to talk to the network
     - --share=network
     # Need to read our config from somewhere
     - --filesystem=xdg-config/input-leap
-    # HACK: so we can put our EIS socket somewhere accessible
-    # This won't be needed once the portal actually supports our interfaces
-    - --filesystem=xdg-run/eis-0
     # Let's show off :)
```

Comment:
This comment needs to be removed, it was there to show that we can run `input-leapc`/`input-leaps` without any X or wayland connection but that doesn't apply here since we need the GUI.

Reply:
removed/fixed